terminal/command.go

```diff
@@ -578,37 +587,52 @@ func breakpoints(t *Term, args string) error {
 	return nil
 }
 
-func setBreakpoint(t *Term, tracepoint bool, argstr string) error {
-	args := strings.Split(argstr, " ")
-	if len(args) < 1 {
-		return fmt.Errorf("address required, specify either a function name or <file:line>")
+func validBreakpointName(name string) bool {
```

Comment:
We shouldn't have to duplicate this logic here, can't we just rely on this error being caught later on down the line?